backends/pipeline_api_test.py

```diff
@@ -209,12 +215,34 @@ def test_preferred_language(self, mocked_get_json):
             mocked_content['language_proficiencies'] = proficiencies
             mocked_get_json.return_value = mocked_content
             pipeline_api.update_profile_from_edx(
-                edxorg.EdxOrgOAuth2, self.user, {'access_token': 'foo_token'}, True)
+                edxorg.EdxOrgOAuth2(strategy=mock.Mock()), self.user, {'access_token': 'foo_token'}, True)
 
             self.user_profile.refresh_from_db()
             assert self.user_profile.preferred_language is None
             assert self.user_profile.edx_language_proficiencies == proficiencies
 
+    def test_login_redirect_dashboard(self):
+        """
+        A student should be directed to /dashboard
+        """
+        student = UserFactory.create()
+        session_set = mock.Mock()
+        backend = edxorg.EdxOrgOAuth2(strategy=mock.Mock(session_set=session_set))
```

Comment:
@noisecapella Is their test for [``not is_new``](https://github.com/mitodl/micromasters/blob/cb4725fc85a6632b9d7862e6875b09a60de40343/backends/pipeline_api.py#L47)  user. What behavior the get?